core/auth/webidentity.go

```diff
@@ -0,0 +1,84 @@
+package auth
+
+import (
+	"context"
+
+	"flamingo.me/flamingo/v3/framework/web"
+)
+
+type (
+	// WebIdentifier resolves an identity from a web request
+	WebIdentifier interface {
+		Identify(ctx context.Context, request *web.Request) Identity
+	}
+
+	// WebAuthenticater returns the first usable web.Result to return to authenticate the user
+	WebAuthenticater interface {
+		Authenticate(ctx context.Context, request *web.Request) web.Result
+	}
+
+	// WebIdentityService calls one or more identifier to get all possible identities of a user
+	WebIdentityService struct {
```

Comment:
Maybe we may add possibility to set default identityProvider in config - in general, anywhere where there are multiple possibilities for authentication, default login page is shown (like keycloak or any other), beside having additional posibilities to login via facebook/google. So, maybe adding IdentifyForDefault/AuthenticateForDefault would be useful.

Reply:
First one is default